fyle_accounting_mappings/views.py

```diff
@@ -198,8 +206,15 @@ def get(self, request, *args, **kwargs):
                 **filters, workspace_id=self.kwargs['workspace_id']
             ).count()
         else:
+            filters = {}
+
+            if active and active.lower() == 'true':
+                filters['source__active'] = True
+            elif active and active.lower() == 'false':
+                filters['source__active'] = False
+
             mapped_attributes_count = Mapping.objects.filter(
-                source_type=source_type, destination_type=destination_type, workspace_id=self.kwargs['workspace_id']
+                source_type=source_type, destination_type=destination_type, workspace_id=self.kwargs['workspace_id'], **filters
```

Comment:
move `source_type` `destination_type` `workspace_id` to the filters dict 
same as line 259